cognoml/tests/unit_tests/test_figshare.py

```diff
@@ -0,0 +1,12 @@
+import os
+import re
+from cognoml import figshare
+
+def test_get_article_versions():
+    article_url = 'https://api.figshare.com/v2/articles/3487685/versions'
+    url_pattern = re.compile("https://api.figshare.com/v2/articles/([0-9]*)/versions/([0-9]*$)", re.MULTILINE)
+    version_to_url = figshare.get_article_versions(3487685)
+    assert type(version_to_url) == dict
```

Comment:
Should we do

```python
assert isinstance(version_to_url, dict)
```

so if `version_to_url` is an OrderedDict (which makes sense here), this won't fail?